doc/source/development/contributing_docstring.rst

```diff
@@ -616,7 +616,7 @@ be added with blank lines before and after them.
 
 The way to present examples is as follows:
 
-1. Import required libraries (except ``numpy`` and ``pandas``)
```

Comment:
leave this, actually generally we *do* want to use double-backticks around pandas as it highlites it

Reply:
> But, I would say we shouldn't convert _all_ occurrences of "`pandas`" to "pandas". IMO, it if it clearly about the _package_ (and not the project in general), it is fine to keep using `pandas` in some places, instead of having a hard rule.

Noted. I'll update my PR to change this.